packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -143,19 +140,29 @@ describe.each( [
 			'Click for Delicious Gnocchi'
 		);
 
-		await user.hover( firstRadio );
+		await hover( firstRadio );
 
-		const tooltip = await screen.findByText(
-			'Click for Delicious Gnocchi'
-		);
+		const tooltip = await screen.findByRole( 'tooltip', {
+			name: 'Click for Delicious Gnocchi',
+		} );
 
 		await waitFor( () => expect( tooltip ).toBeVisible() );
 
-		await cleanupTooltip( user );
+		// hover outside of radio
+		await hover( document.body );
+		await hover( document.body, { clientX: 10, clientY: 10 } );
+
+		// Tooltip should hide
+		await waitFor( () =>
+			expect(
+				screen.queryByRole( 'tooltip', {
+					name: 'Click for Delicious Gnocchi',
+				} )
+			).not.toBeInTheDocument()
```

Comment:
Is `waitForElementToBeRemoved()` a better possible alternative here?

Reply:
It is not, because it turns out that there is no need to wait for it — the tooltip gets removed immediately, so I refactored the code and removed the `waitFor` altogether. I also took the chance to refine some tooltip-related tests in `ToggleGroupControl` to actually test that the tooltip doesn't show even after its default delay has passed https://github.com/WordPress/gutenberg/pull/57547/commits/428809ba6a87e1bc9a263ee09fe42e64edffe74c